plugin/trino-raptor-legacy/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <artifactId>trino-raptor-legacy</artifactId>
     <packaging>trino-plugin</packaging>
-    <description>Trino - Raptor Legacy Connector</description>
+    <description>Trino - Raptor legacy connector</description>
```

Comment:
I am honestly not sure why we still drag this along .. is it used anywhere.. but also thats an unrelated question

Reply:
Haha, I'm asking this myself every day :P